### ISSUE TITLE
Have local references in README.md

### ISSUE THREAD

**Issue by chevillotn** [UVVM/UVVM#76]
Hi,
As we are using your UVVM library locally to our project, we would like to have local references in the README.md instead of a reference to the GitHub with no tag.
The code in question is for example:
```
UVVM consists currently of the following elements:
- [Utility Library](https://github.com/UVVM/UVVM/blob/master/uvvm_util/README.md)
```
it could be replaced with:
```
UVVM consists currently of the following elements:
- [Utility Library](./README.md)
```

This does not point to the release in question.
Thanks.

**Comment by UVVM** [comment]
Hi,

Thanks for the observation, we will change this in our next release.

Best regards,
Erick

**Comment by UVVM** [comment]
Fixed in release v2019.09.02
